src/features/messages/MessageSearch.tsx

```diff
@@ -96,7 +140,16 @@ export function MessageSearch() {
             onChangeEndTimestamp={setEndTimeFilter}
           />
         </div>
-        <Fade show={!isAnyError && isValidInput && isAnyMessageFound && !!multiProvider}>
+        <Fade
+          show={
+            !isAnyError &&
+            isValidInput &&
+            isValidOrigin &&
+            isValidDestination &&
+            isAnyMessageFound &&
```

Comment:
Long list, should we make a var above to combine these?

Reply:
created a `showMessageTable` variable to combine these